target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mf289f.dts

```diff
@@ -0,0 +1,427 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2022, Pawel Dembicki <paweldembicki@gmail.com>.
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "ZTE MF289F";
+	compatible = "zte,mf289f";
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+		};
+
+	chosen {
+		/*
+		 * bootargs forced by u-boot bootipq command:
+		 * 'ubi.mtd=rootfs root=mtd:ubi_rootfs rootfstype=squashfs rootwait'
+		 */
+		bootargs-append = " root=/dev/ubiblock0_1";
+		};
+
+	/* This node is used to restart modem module to avoid anomalous behaviours on initial communication  */
+	gpio-restart {
+		compatible = "gpio-restart";
+		gpios = <&tlmm 8 GPIO_ACTIVE_HIGH>;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led_status: led-status {

```

Comment:
Need new line here.  And I see whole wifi led has been droped? Sorry I didn't make it clear earlier, we just need to remove the `led_wlan: ` string. f4f22098f390d6dc62989bcc8a65400e90cb2695

Reply:
what do you mean for "New line"

it should be:

```
leds {

		compatible = "gpio-leds";
		led_status: led-status {
```

instead of

```
leds {
		compatible = "gpio-leds";
		led_status: led-status {
```

Also other stuff doesn't have new line after first parenthesis. Do I need to add on the whole DTS?
Don't know why all this errors, base DTS was from ZTE-MF286D, except for the tabs the structure is pretty similar

Let me know, just to avoid too many resquash, thx :)

Reply:
> Need new line here. And I see whole wifi led has been droped? Sorry I didn't make it clear earlier, we just need to remove the `led_wlan: ` string. [f4f2209](https://github.com/openwrt/openwrt/commit/f4f22098f390d6dc62989bcc8a65400e90cb2695)

regarding led_wlan, do you mean in this way:

```
                led-wlan {
                        function = LED_FUNCTION_WLAN;
                        color = <LED_COLOR_ID_BLUE>;
                        gpios = <&tlmm 61 GPIO_ACTIVE_LOW>;
                        linux,default-trigger = "phy0tpt";
                };
```

?

thx!

Reply:
should be 
```
leds {
		compatible = "gpio-leds";

		led_status: led-status {
```

> regarding led_wlan, do you mean in this way:
> 
> ```
>                 led-wlan {
>                         function = LED_FUNCTION_WLAN;
>                         color = <LED_COLOR_ID_BLUE>;
>                         gpios = <&tlmm 61 GPIO_ACTIVE_LOW>;
>                         linux,default-trigger = "phy0tpt";
>                 };
> ```
> 
> ?
> 
> thx!

Yes, and please also notice space problem. Hope you don't feel I'm too verbose, but some main maintainers may be strict about formatting.